idaes/gas_solid_contactors/properties/oxygen_iron_OC_oxidation/gas_phase_thermo.py

```diff
@@ -489,8 +490,12 @@ def _dens_mol(self):
                             units=pyunits.mol/pyunits.m**3)
 
         def ideal_gas(b):
-            return (b.dens_mol*b._params.gas_const*b.temperature*1e-2 ==
-                    b.pressure)
+            return (
+                    b.dens_mol*
+                    b._params.gas_const*(1e3*pyunits.J/pyunits.kJ)
```

Comment:
You should use the IDAES Constants class for this, along with a unit conversion if required.

`from idaes.core.util.constants import Constants`